lib/Doctrine/ruleset.xml

```diff
@@ -47,6 +47,7 @@
                     close => closedir,
                     delete => unset,
                     doubleval => floatval,
+                    extract => list,
```

Comment:
This should not suggest a replacement here, list is not a function (also we use `[...]` instead of `list(...)`) so replacement suggestion is not correct imho.

Reply:
Indeed, as https://php.net/list says:
```
Like array(), this is not really a function, but a language construct. list() is used to assign a list of variables in one operation.
```

Thanks, gonna remove it :+1: